torch/csrc/inductor/inductor_ops.cpp

```diff
@@ -0,0 +1,27 @@
+#include <ATen/ops/matmul.h>
+#include <torch/csrc/inductor/inductor_ops.h>
+#include <torch/library.h>
+
+namespace torch {
+namespace inductor {
+using namespace at;
+
+Tensor _mm_plus_mm(
+    const Tensor& a,
+    const Tensor& b,
+    const Tensor& c,
+    const Tensor& d,
+    Tensor& out) {
+  at::matmul_out(out, a, b);
+  out.addmm_(c, d);
```

Comment:
Why are you using matmul for one and mm (or addmm) for the other one? We should use either mm for both or matmul for both. I think using `mm` for both would be best here.